controller/emulator.py

```diff
@@ -69,21 +69,17 @@ def start(version, wipe):
     path = ROOT_DIR / "firmware/bin"
 
     if version[0] == "2":
-        profile = "/var/tmp/trezor.flash"
-        if wipe and os.path.exists(profile):
-            os.remove(profile)
+        model_t_image = "/var/tmp/trezor.flash"
```

Comment:
Let's keep the word `profile`, we use it internally. So for example `model_t_profile`? Or `profile_tt`?

Reply:
Yes, will change that